contracts/governance/extensions/GovernorTimelockControl.sol

```diff
@@ -60,9 +60,15 @@ abstract contract GovernorTimelockControl is IGovernorTimelock, Governor {
         bytes32 queueid = _timelockIds[proposalId];
         if (queueid == bytes32(0)) {
             return currentState;
-        } else if (_timelock.isOperationDone(queueid)) {
+        }
+
+        TimelockController.OperationState _operationState = _timelock.getOperationState(queueid);
+        if (_operationState == TimelockController.OperationState.Done) {
             return ProposalState.Executed;
-        } else if (_timelock.isOperationPending(queueid)) {
+        } else if (
+            _operationState == TimelockController.OperationState.Pending ||
+            _operationState == TimelockController.OperationState.Ready
+        ) {
```

Comment:
The issue with this change is that the new governors that may be deployed in the future are not going to be compatible with the old TimelockController that have been deployed for a while.

Reply:
Retrocompatibility is something kept even in breaking cahnges?

Reply:
Not everything needs to be kept, but I would argue that allowing new governor to use existing TimelockController is important.